Katas/FizzBuzzTest/Changelog.md

```diff
@@ -17,4 +17,6 @@
 *Update: test case for 10 and minimum passing code.
 *Update: test case for 25 and minimum passing code.
 *Note: refactoring now due to rule of 3.
-*Update: refactored implimnetation
\ No newline at end of file
+*Update: refactored implimnetation
+*Update: test case for 15 and minimum passing code.
+*Note: I am not happy with the jump to put it at the top of the function but i hope any comments will help.
```

Comment:
what's your concern with having done that?

Reply:
I dont know i felt like knowing that my code would fail by starting with it at the bottom of the function felt almost like cheating as there was no way the test i wrote could drive that knowledge. Is that wrong?